wpilibj/src/main/java/edu/wpi/first/wpilibj/experimental/controller/PIDControllerRunner.java

```diff
@@ -0,0 +1,114 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.experimental.controller;
+
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.function.DoubleConsumer;
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj.Notifier;
+
+public class PIDControllerRunner {
```

Comment:
I think this class should implement `Sendable` so dashboards can enable/disable the runner, like the current PIDController implementation

Reply:
@calcmogul This is still relevant. We lose feature parity versus the old implementation by not allowing the PID runner to be controlled from a dashboard

Reply:
Pushed a change to fix this; the runners are now Sendable.